### ISSUE TITLE
not converge

### ISSUE THREAD

**Issue by liuhengli** [LevinJ/SSD_tensorflow_VOC#6]
Soory, I using this code train in voc2007 and voc2012 dataset, the total loss is 5 can't decrease。Can you tell some ways?

**Comment by LevinJ** [comment]
Hmm, that's a bit strange. Did you change any codes? Also can you post your loss chart here? You can get the loss chart via tensorboard.

**Comment by liuhengli** [comment]
@LevinJ I am only modify some dataset dir .
![2017-09-12 20-57-30](https://user-images.githubusercontent.com/11975469/30327049-3b211c36-97fd-11e7-9807-fbd1965b0ff8.png)

**Comment by LevinJ** [comment]
Hi @liuhengli , this is probably what is expected. Take a look at the loss chart I has in readme, yours is pretty similar as mine, except that you might need to train a bit longer to reduce the loss a bit further.

As a side note, I added dropout layer in the net to prevent overfitting, so the loss may not always drop, instead it will plateau at some point.

**Comment by liuhengli** [comment]
ok, Think you very much, maybe I am too impatient

**Comment by LevinJ** [comment]
Yeah, the training currently does take a long time, about 58 hours on my Nvidia GTX 1080 GPU.
